float64 x
float64 y
string coord
int64 planet
float64 off_x
float64 off_y
string offcoord
string hosei
float64 lamda
int64 dcos
float64 vel_x
float64 vel_y
float64 movetime
bool limit
float64 time